Library/String+Currency.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import UIKit.UIColor
+import UIKit.UIFont
+
+public func attributedCurrencyString(
```

Comment:
Oh, something else I wanted to suggest, seeing as this is not actually an extention on `String` should the filename be `String+Currency.swift`? This may be fine to place in `SharedFunctions`? 🤷‍♂